tools/db_crashtest.py

```diff
@@ -118,6 +118,7 @@
     "optimize_filters_for_memory": lambda: random.randint(0, 1),
     "partition_filters": lambda: random.randint(0, 1),
     "partition_pinning": lambda: random.randint(0, 3),
+    "get_update_since_one_in": lambda: random.choice([0, 0, 0, 1000]),
```

Comment:
May need to sanity check that disable_wal is 0.

Reply:
Fixed